getting_started/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -300,6 +300,7 @@ states and quick structs:
 
     # Same example, lua-style support.
     # This syntax is a lot more readable and usable
+    # Like any GDScript identifier, keys written in this form cannot start with a digit.
```

Comment:
This isn't actually correct. I've been able to use digits as key names before, see [here](https://github.com/MichiRecRoom/discord.gd/blob/6f6b71292229e1c400954576178532c3bcf3567f/addons/discord.gd/gateway/connection.gd#L19).